internals/secrethub/path_placeholders.go

```diff
@@ -0,0 +1,10 @@
+package secrethub
+
+const (
+	repoPathPlaceHolder                  = "<namespace>/<repo>"
+	dirPathPlaceHolder                   = repoPathPlaceHolder + "/<dir>[/<dir> ...]"
+	optionalDirPathPlaceHolder           = repoPathPlaceHolder + "[/<dir> ...]"
+	secretPathPlaceHolder                = optionalDirPathPlaceHolder + "/<secret>"
+	secretPathOptionalVersionPlaceHolder = secretPathPlaceHolder + "[:<version>]"
+	anyPathPlaceHolder                   = "<namespace>[/<repo>[/<dir> ...][/<secret>[:<version>]]]"
```

Comment:
Though this is technically correct, I don't think it makes things a lot more understandable. Can we make this simpler?

Reply:
How about just using `secrethub ls [<path>]`?

Reply:
Sounds a lot better :+1: